docs/nodes/validator-node.md

```diff
@@ -203,22 +203,18 @@ cd $HOME
 rm -rf celestia-node
 git clone https://github.com/celestiaorg/celestia-node.git
 cd celestia-node/
-APP_VERSION=$(curl -s \
-  https://api.github.com/repos/celestiaorg/celestia-node/releases/latest \
-  | jq -r ".tag_name")
-git checkout tags/$APP_VERSION -b $APP_VERSION
 make install
 ```
 
 Verify that the binary is working and check the version with `celestia version` command:
 
 ```sh
 $ celestia version
-Semantic version: v0.2.0
-Commit: 1fcf0c0bb5d5a4e18b51cf12440ce86a84cf7a72
-Build Date: Fri 04 Mar 2022 01:15:07 AM CET
+Semantic version: v0.3.0-rc1-7-g45f23d8
```

Comment:
Can you delete the instructions in `Install Celestia Node` and just link them to the tutorial on installation?

Can just add this line:
```md
You can follow the tutorial for installing Celestia Node [here](../../developers/celestia-node)
```

Reply:
i think i did it